### ISSUE TITLE
Явка с клиента не меняет значение явки на сервере

### ISSUE THREAD

**Issue by kirzas** [sedovalx/taxi#103]
Поле "явка" в форме "Payment" (новый платеж) сейчас не меняет значение в колонке Явка в КС.

Начат разговор в https://github.com/sedovalx/taxi/issues/86#issuecomment-121968062

**Comment by sedovalx** [comment]
Странно, все менялось. Условия следующие - если создается новый платеж по **аренде**, и в нем указана галка явки, в день прохождения этого платежа в столбце явки в КС будет стоять галка.

**Comment by sedovalx** [comment]
@Argelein не воспроизводится. У меня все меняется:
- выбрал активную аренду в КС
- нажал "Создать платеж"
- на форме операции выбрал тип "Арендная плата", указал галку явки и сумму
- нажал "Сохранить"
- в КС отобразилась галка

**Comment by sedovalx** [comment]
Да, я понял, почему не работает. Явка рассчитывается, исходя из той даты, что была передана на сервер из кассовой формы. Ранее это была всегда текущая дата, и явка рассчитывалась на текущую дату. Сейчас после #92 на сервер стала передаваться будущая дата, и явка соответственно стала рассчитываться, исходя из нее. Т.к. на будущую дату нет операций, то и в кассовой форме в столбце явки пусто. Нужно думать, как это обходить.

Можно рассчитывать явку всегда относительно текущей серверной даты, независимо от того, какая дата была передана из кассовой формы. Это по идее должно сработать. Но тут возникает проблема - если пользователь захочет посмотреть состояние кассовой формы на какой-то момент из прошлого (нужно ли это?), то столбец явки будет отображаться неверно - он все равно будет считаться относительно текущей даты.

Можно считать явку относительно меньшей из двух дат - текущей и переданной из кассовой формы.

В общем, что-то мне все это непонятно. @hodkoff2 @Argelein  что думаете?
